app/interface/http/handlers.go

```diff
@@ -0,0 +1,54 @@
+package http
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/mailgun/catchall"
+
+	"github.com/xfoxawy/catchall-domain/app/repository"
+)
+
+type Handlers struct {
+	repo *repository.EventsCounterRepository
+}
+
+func NewHttpHandlers(repo *repository.EventsCounterRepository) *Handlers {
+	return &Handlers{repo: repo}
+}
+
+func (h *Handlers) GetDomainStatusHandler(ctx *gin.Context) {
+	var err error
+	domain := ctx.Param("domain")
+	decounter, err := h.repo.FindOneByDomain(ctx, domain)
+	if err != nil {
+		ctx.AbortWithError(500, err)
+	}
+	if decounter.Bounced > 0 {
```

Comment:
why not execute a `select count` that would save some bandwidth of data transfer